### PR TITLE
Add grey bar to top of archived threads

### PR DESCRIPTION
**Pull request by sjuraschka**
I added a grey bar to the top of archived threads to make them more visually distinct from open threads.

This also helps users distinguish when they are in the inbox vs. when they are not.
screen shot 2016-06-01 at 5 16 08 pm

Fixes https://github.com/braidchat/meta/issues/449 and https://github.com/braidchat/meta/issues/451

### CODE REVIEW

File: src/braid/ui/styles/vars.cljs [hunk 1]
```diff
@@ -18,3 +18,9 @@
 (def dark-bg-color "#3A4767")
 
 (def border-radius (px 3))
+
+(def private-thread-accent-color "#5f7997")
+
+(def limbo-thread-accent-color "#CA1414")
+
+(def archived-thread-accent-color "#AAAAAA")
```
Comment (jamesnvc) [inline src/braid/ui/styles/vars.cljs]: should be a single newline at the end of the file (note the little red warning thing github is showing)

**Comment by jamesnvc** [comment]
Looks great!

File: src/braid/ui/styles/vars.cljs (hunk 1 above)
Comment (rafd) [inline src/braid/ui/styles/vars.cljs]: this is my bad, I got confused re: your comment on: https://github.com/braidchat/braid/pull/17/files/d7a8a84a3b4916bff855f7ee4c6f6e6f489fc640#r65015137 b/c that commit *did* remove trailing new lines, and so I thought you meant that you shouldn't have any newlines at the EOF

Comment (jamesnvc) [inline src/braid/ui/styles/vars.cljs]: Yeah, sorry - should be exactly one newline at the end, but not more than one - my comment on that other pull request was objecting to the three newlines

Comment (jamesnvc) [inline src/braid/ui/styles/vars.cljs]: Sorry for not being clear!

**Pull request merged**